src/containers/Accounts/Login.js

```diff
@@ -93,7 +93,7 @@ class Login extends Component {
   }
 
   render() {
-    const { intl } = this.props;
+    const { intl, location } = this.props;
```

Comment:
What's the meaning of the variable name `location` here?

Reply:
`react-router` provides `location` which can be used to store state when switching routes. This is being used to check if there was a callback error that put the user back to this screen.